CapstoneProject/Models/ActivityRec.m

```diff
@@ -0,0 +1,12 @@
+//
+//  ActivityRec.m
+//  CapstoneProject
+//
+//  Created by Apoorva Chilukuri on 7/14/22.
+//
+
+#import "ActivityRec.h"
+
+@implementation ActivityRec
```

Comment:
Let's use the full name for the class name. In this case, it should be called `ActivityRecommendation`.